ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -694,6 +694,64 @@ literals:
   - PRESENT
   - ABSENT
 
+# Added for TPKD day2
+- condensing_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- condensing_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_supply_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- chilled_return_water_isolation_valve_mode:
+  - MANUAL
+  - AUTO
+- oil_heater_status:
+  - ON
+  - OFF
+- inhibit_start_time_sensor
+- load_limit_capacity
+- total_compressor_count
+- compressor_run_time_sensor
+- guidevane_percentage_command
+- oil_temperature_sensor
+- oil_pump_differential_pressure_sensor
+- exhaust_gas_compressor_temperature_sensor
+- bearing_temperature_sensor
+- winding_temperature_sensor
+- compressor_temperature_setpoint
+- load_compressor_percentage_setpoint
```

Comment:
What does this mean? Compressors don't deal in weight. Can you please explain what this is?

Reply:
The definition of load has been changed

Reply:
Still don't understand. Explain what this field is.

Reply:
If this is meant to be a target cooling load, use `cooling_thermal_power_setpoint`. Not sure what the point of compressor is in this; does each compressor have its own setpoint?

Reply:
load_compressor_percentage_setpoint (NO1 Compressor load percentage point) would describe the chiller compressor current power output against the rated power output in percentage.

Reply:
i'm still confused. is this a setpoint or a command? generally you do not give chillers a cooling load setpoint, and you usually do not tell a compressor how hard to run by sending it a power setpoint. could you explain how this controls in more detail?

Reply:
This point has been removed